week7/cooper_client_react/04_adding_a_user_class.md

```diff
@@ -16,26 +16,39 @@ Run the following command for an easy one-step installation.
 
 `$ rails g devise_token_auth:install User auth`
 
-Remember to migrate your database in order to create the `users` table (the Devise generator created a migration for you). But before you do that, make sure to open up the migration file and change the data type for `tokens` to `text`.
+Make sure to open up the migration file that was generated and add some more columns that are needed in order for Deivse token auth to work properly with our client later. Please add this:
 ```
 #db/migrate/XXX_devise_token_auth_create_users.rb
-## Tokens
-t.text :tokens
+## The columns we want to add
+t.integer :sign_in_count, default: 0
+t.datetime :current_sign_in_at
+t.datetime :last_sign_in_at
+t.string :current_sign_in_ip
+t.string :last_sign_in_ip
 ```
 
+We also want to remove some of the columns that Devise token auth has added to that migration file. There is a section in th migration file that says `## User info`, please remove these columns:
+```
+t.string :name
+t.string :nickname
+t.string :image
+```
+
+At the moment we are only building the MVP of the this application, so adding for instance `:nickname` would be out of scope and that `User` attribute would not have been used.
```

Comment:
Not sure about this sentence... It comes across a bit unclear.

Reply:
rephrased it